### DISCUSSION TITLE
OpenStreetMap instead of Mapbox?

### DISCUSSION THREAD

**Discussion by BabyFnord** [firefly-iii/firefly-iii#5837] [General]
Hello @JC5 and everyone else,

I've come across the red highlighted notice a few times, about supplying an API-key for the Mapbox feature. While there is a free account option at Mapbox, in terms of privacy my trust is with FOSS projects, not with companies who likely got an idea for how to generate revenue with data from "free accounts".

In short: Would it be a feature worth integrating at some point, to use [OpenStreetMap](https://www.openstreetmap.de) instead of proprietary Mapbox? If not, well, kudos anyway for keeping FireFly III FOSS <3 !

Thank you for considering,
Patrick

**Comment by JC5** [comment]
Yes, this is on my long-term list to switch out. At the time, there was no proper library from openstreetmap. These days there is. :+1:
